dev_docs/tutorials/adding_performance_metrics.mdx

```diff
@@ -171,7 +171,12 @@ as stable and reproducible as possible.
 
 #### Machine specifications
 
-The machine has these [specs](https://www.hetzner.com/dedicated-rootserver/ex100).
+All benchmarks are run on bare-metal machines with the [following specifications](https://www.hetzner.com/dedicated-rootserver/ex100):
+
+CPU: Intel® Core™ i9-12900K
+RAM: 128 GB
+SSD: 1.92 TB Datacenter Gen4 NVMe 
+All machines are connected via a dedicated 1 GBit switched network.
```

Comment:
There is only one machine on which everything is running so maybe remove that info to avoid that readers think the Ethernet connection is used by benchmarks?